tests/scripts/github_docs_comment.py

```diff
@@ -25,11 +25,22 @@
 from git_utils import git, GitHubRepo, parse_remote
 from cmd_utils import init_log
 
+DOCS_BOT_MARKER = "<!---docs-bot-comment-->"
```

Comment:
```suggestion
DOCS_BOT_MARKER = "<!---docs-bot-comment-->\n\n"
```

I tried it out manually in [this comment](https://github.com/apache/tvm/pull/11909#issuecomment-1167891541) and it apparently busts the parsing on GitHub, some newlines fixes it up though

Reply:
Interesting, I made the change.